### PR TITLE
Add Get-OpenADPrincipalGroupMembership cmdlet

### PR DESCRIPTION
**Pull request by theaquamarine**

### CODE REVIEW

**Comment by theaquamarine** [comment]
I think the failed [Fedora_37_Heimdal](https://github.com/jborean93/PSOpenAD/actions/runs/4639504062/jobs/8210426804) test just needs a rerun, it failed installing PowerShell (and has run successfully [elsewhere](https://github.com/theaquamarine/PSOpenAD/actions/runs/4639433882/jobs/8210276190))

**Comment by jborean93** [comment]
Thanks for the new addition, just a minor question on the doc example and a stray backtick that needs to be removed.

File: docs/en-US/Get-OpenADPrincipalGroupMembership.md [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+PS C:\> Get-OpenADPrincipalGroupMembership 'Administrator' -Recursive
+```
+
+List the groups that Administrator is a member of, directly or indirectly.
+
+## PARAMETERS
+
+### -AuthType
+The authentication type to use when creating the `OpenAD` session.
+This is used when the cmdlet creates a new connection to the `-Server` specified`.
```
[the remark this answers is not in the document]
Comment (theaquamarine) [reply docs/en-US/Get-OpenADPrincipalGroupMembership.md]: Removed!

**Comment by theaquamarine** [comment]
Updated the example's description to be clear it's referring to a computer, as well as adding the `$`.

**Review by jborean93 — APPROVED** [review]

**Comment by jborean93** [comment]
Thanks for working on this.

**Pull request merged**